AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_DirectionalLightAdded.py

```diff
@@ -146,37 +325,452 @@ def AtomEditorComponents_DirectionalLight_AddedToEntity():
+
+        directional_light_component.set_component_property_value(
+            AtomComponentProperties.directional_light('Blend between cascades'), False)
+        Report.result(
+            Tests.blend_between_cascades_back_on,
+            directional_light_component.get_component_property_value(
+                AtomComponentProperties.directional_light('Blend between cascades')) is False)
+
+        # 30. Set Shadow filter method to ESM, PCF+ESM, then back to default of None.
+        directional_light_component.set_component_property_value(
+            AtomComponentProperties.directional_light(
+                'Shadow filter method'), DIRECTIONAL_LIGHT_SHADOW_FILTER_METHOD["ESM"])
+        Report.result(
+            Tests.shadow_filter_method_set_to_ECM,
+            directional_light_component.get_component_property_value(
+                AtomComponentProperties.directional_light(
+                    'Shadow filter method')) == DIRECTIONAL_LIGHT_SHADOW_FILTER_METHOD["ESM"])
+
+        directional_light_component.set_component_property_value(
+            AtomComponentProperties.directional_light(
+                'Shadow filter method'), DIRECTIONAL_LIGHT_SHADOW_FILTER_METHOD["PCF+ESM"])
+        Report.result(
+            Tests.shadow_filter_method_set_to_ECM_and_PCF,
+            directional_light_component.get_component_property_value(
+                AtomComponentProperties.directional_light(
+                    'Shadow filter method')) == DIRECTIONAL_LIGHT_SHADOW_FILTER_METHOD["PCF+ESM"])
+
+        directional_light_component.set_component_property_value(
+            AtomComponentProperties.directional_light(
+                'Shadow filter method'), DIRECTIONAL_LIGHT_SHADOW_FILTER_METHOD["None"])
+        Report.result(
+            Tests.shadow_filter_method_set_to_None,
+            directional_light_component.get_component_property_value(
+                AtomComponentProperties.directional_light(
+                    'Shadow filter method')) == DIRECTIONAL_LIGHT_SHADOW_FILTER_METHOD["None"])
+
+        # 31. Turn Fullscreen Blur off then back on.
+        directional_light_component.set_component_property_value(
+            AtomComponentProperties.directional_light('Fullscreen Blur'), False)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.fullscreen_blur_off,
+            directional_light_component.get_component_property_value(
+                AtomComponentProperties.directional_light('Fullscreen Blur')) is False)
+
+        directional_light_component.set_component_property_value(
+            AtomComponentProperties.directional_light('Fullscreen Blur'), True)
+        Report.result(
+            Tests.fullscreen_blur_back_on,
+            directional_light_component.get_component_property_value(
+                AtomComponentProperties.directional_light('Fullscreen Blur')) is True)
+
+        # 32. Fullscreen Blur Strength can be set to 7.0 then back to the default of 2.5.
```

Comment:
is this value correct? the docstring also mentions this; `32) Fullscreen Blur Strength can be set to 7.0 then back to the default of 2.5`
but in the code down below it looks like it sets it to 0 and then to 0.667 for the directional light component's "Fullscreen Blur Strength" property

should update the docstring and comment to match these values

Reply:
Good catch, fixed the docstring